### DISCUSSION TITLE
Help modifying Airsim API

### DISCUSSION THREAD

**Discussion by mpalin** [microsoft/AirSim#3431] [Support Q&A] [answered]
I am trying to modifying the return results of the getMultirotorState() function. My project is simulating a gps disabled environment  and will be distributed to several teams. I am trying to prevent the teams from using gps by disabling the gps_location from the return of client.getMultirotorState().

I have disabled the gps as a default sensor, but the gps_location attribute of MultirotorState still shows a valid value in the function return.

**Comment by mpalin** [comment, marked as answer]
I solved this by setting the value to zero in my Airsim install and then rebuilding and copying the new plugin folder into my project

**Reply by ahmed-elsaharti** [reply]
you might need to also do the same with the getGpsData API **if** its not affected by you disabling the default gps sensor.
